examples/add_fiducials_for_defacing.py

```diff
@@ -0,0 +1,119 @@
+"""
+================================
+Add fiducial points for Defacing
+================================
+
+In this example, we will add fiducial points so that mne-bids
+knows where the face is for the defacing anonymization algorithm.
+"""
+
+# Authors: Alex Rockhill <aprockhill206@gmail.com>
+# License: BSD (3-clause)
+
+###############################################################################
+# We are importing everything we need for this example:
+
+import os
+import os.path as op
+import importlib
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.io import loadmat
+
+from nilearn.plotting import plot_anat
+
+import mne
+from mne.datasets import sample
+
+from mne_bids import write_anat
+
+###############################################################################
+# Load in the raw data to which the dig montage will be added.
+
+data_path = sample.data_path()
+
+# set the path of the T1
+t1_mgh_fname = op.join(data_path, 'subjects', 'sample', 'mri', 'T1.mgz')
+
+# load in ECoG data
+mat = loadmat(mne.datasets.misc.data_path() + '/ecog/sample_ecog.mat')
+ch_names = mat['ch_names'].tolist()
+elec = mat['elec']  # electrode positions given in meters
+
+###############################################################################
+# Create unaligned datapoints in a dig montage and then align them to the MRI.
+
+# Now we make a montage stating that the sEEG contacts are in head
+# coordinate system (although they are in MRI). This is compensated
+# by the fact that below we do not specicty a trans file so the Head<->MRI
+# transform is the identity.
+montage = mne.channels.make_dig_montage(ch_pos=dict(zip(ch_names, elec)),
+                                        coord_frame='head',
+                                        lpa=np.zeros((3,)),
+                                        nasion=np.zeros((3,)),
+                                        rpa=np.zeros((3,)))
+
+# Make an info object containing the montage
+info = mne.create_info(ch_names, 1000., 'ecog', montage=montage)
+
+# Use random data as a placeholder
+raw = mne.io.RawArray(np.random.random((len(ch_names), 1000)), info)
+
+# Now align the fiducial points and save out the aligned trans file.
```

Comment:
I don't understand what you need to "align" here because we are working with iEEG

Reply:
yes agreed. You put your finger a current "glitch" of the ECoG example. The .mat file contains points in MRI coords but it pretends it's in head and then uses a trans head<->MRI that is the identity. With make_dig_montage you should say that coord_frame is 'mri' and pass fiducials in MRI coord too.

Reply:
Hmmm.... I am having a difficult time making sense of this. When you are using 'make_dig_montage' where are you getting these fiducial coordinates in MRI space?

In my experience, a patient has intracranial recordings in my case in EDF format with no digitization information. I also have a CT and and aMRI. From the aMRI, I can get a head model and from the CT I can get the positions of the electrodes (elec in the mat file in this example). Where I am really lost is where are you getting any information about the nasion, LPA and RPA? In my case, which I assume is typical but maybe not, there is no information about where these points are in headspace or ecog space or any other space. I am really confused by these "pass fiducials in MRI coord" and "save the anatomical landmarks directly," I don't know where you would get this information.